README.md

```diff
@@ -58,8 +58,8 @@ jobs:
       with:
         args: |
           --all \
-          --target /data \
-          --docker-hub userspace-name
+          --target addon-folder \
+          --docker-hub ${{ secrets.DOCKERHUB_USERNAME }
```

Comment:
Your DockHub username doesn't have to match the space one is publishing in?
For example; my Docker account can publish to multiple spaces.

Reply:
Right again. Now I tried some middle-ground with `user-name-or-space-name`.
Examples are mainly for newbees like me, hard-core devs like you please fine-tune your perfectionism toward our level.